src/main/scala/io/citrine/lolo/validation/Merit.scala

```diff
@@ -96,7 +96,7 @@ case class StandardError(rescale: Double = 1.0) extends Merit[Double] {
   * In the absence of a closed form for that coefficient, it is model empirically by drawing from N(0, x) to produce
   * an "ideal" error series from which the correlation coefficient can be estimated.
   */
-case object UncertaintyCorrelation extends Merit[Double] {
+case class UncertaintyCorrelation(rng: Random = Random) extends Merit[Double] {
```

Comment:
Likewise, rng could be added to `evaluate`'s signature.

Reply:
Was this changed to a class to  allow for `rng` or to allow for Merits to have parameters (such as including a coverage level in the standard confidence)?

Reply:
This was changed to allow for rng, and I flipped the others for uniformity. That's reverted now, as `rng` is now passed in `evaluate`'s signature.